routes/productRouter.js

```diff
@@ -4,9 +4,11 @@ const productController = require('../controllers/productController');
 const router = express.Router();
 
 router.get("/main", productController.getRandomProducts);
+router.get("/nav", productController.getCategories);
```

Comment:
```suggestion
router.get("/random", productController.getRandomProducts);
router.get("/categories", productController.getCategories);
```

- 위와 같이 데이터 리소스의 성질을 나타낼 수 있게 수정해주세요.

Reply:
- 가장 이상적인 구조는 프론트엔드에서 products를 뽑아오는 형태의 엔드포인트에 요청을 보내면, 백엔드는 해당 내용에 random 쿼리 빌딩과 offset limit 값 설정을 통해서 원하는 개수의 랜덤 값만 보내주는 모습이오니, 추후 리팩토링 시 참고해주세요!